index.html

```diff
@@ -222,40 +222,48 @@ <h3 id="rec-track"> Normative Specifications </h3>
+              awaiting satisfaction of the <a href="#success-criteria">Success
+              Criteria</a>, which require two interoperable implementations of
+              the Presentation API.  The Working Group will also await
+              demonstration of interoperability between browsers and devices
+              before moving the specification to Proposed Recommendation.
             </p>
 
             <p>
               The <a href="https://github.com/webscreens/openscreenprotocol">
               Open Screen Protocol</a>, under incubation in
               the <a href="http://www.w3.org/community/webscreens/"> Second
-              Screen Community Group</a>, is one proposed basis for
-              interoperability, and two implementations based on it would
-              satisfy those criteria. The charter timeline provides additional
-              time for interoperable implementations to be completed and the
-              Success Criteria to be met.
+              Screen Community Group</a>, will be the basis for demonstrating
+              interoperability between browsers and devices.  Two
+              implementations based on it would satisfy those criteria. The
+              charter timeline provides additional time for Open Screen Protocol
+              implementations to be completed and the Success Criteria to be
+              met.
             </p>
 
             <p>
-              This Working Group plans to assess progress on interoperability
-              and implementation status regularly, and to request re-chartering
-              according to progress on meeting Success Criteria for the
-              specification.
+              This Working Group plans to meet regularly to assess progress on
+              meeting Success Criteria for the Presentation API and
+              interoperability based on the Open Screen Protocol, and request
+              rechartering as needed.
             </p>
           </dd>
           <dt id="level-2"> Presentation API Level 2 </dt>
           <dd>
-            <p> A second version of the Presentation API that integrates
+            <p>
+              A second version of the Presentation API that integrates
               features the Working Group resolved not to include in the first
               version in the interest of time, and also feedback from Web developers
               on the first version. New features that fall within the scope of
-              this Working Group may first be incubated and matured in an
+              this Working Group will first be incubated and matured in an
               appropriate Community Group.
             </p>
+            <p>
+              No Level 2 feature for the Presentation API will be brought out of
+              incubation until the critera for moving the current Presentation
+              API to Proposed Recommendation have been met.
+            </p>
```

Comment:
These changes, coupled with postponing the publication of the First Public Working Draft of the Level 2 spec until Q4 2019 (one year after the end of this charter), essentially put that spec out of charter for now, and I would assume that W3C Management will just ask us to drop it entirely from the Charter if we adopt these changes. I would rather either:

1. consider that this is not a big deal. After all, this Working Group has been good at avoiding losting time on topics that should be given low priority, and some of the features (the Testing API comes to mind) could be developed and implemented in parallel, perhaps by other teams.
=> Stick to the original text and complete with something along the lines of: "Whenever possible, this Working Group will give priority to progressing the first level of the Presentation API to Recommendation". Set **Q4 2018** as milestone for FPWD
2. Acknowledge that the group will not work on the Level 2 spec this year
=> Drop the Presentation API Level 2 from the charter and complete the description of the Presentation API with something along the lines of: "A second version of the Presentation API that integrates features the Working Group resolved not to include in the first version in the interest of time, and also feedback from Web developers on the first version, will first be incubated and matured in an appropriate Community Group. This Working Group expects to bring such features out of incubation through rechartering, when the criteria for moving the current Presentation API to Proposed Recommendation have been met.

Personally, I prefer the freedom left by option 1, but 2 is ok as well.

Reply:
Good point.

I would prefer version #2, since moving something to FPWD in 2018 commits the group to advancing it in 2019, even if the interop criteria for Level 1 have not yet been met.

If the interop milestone happens ahead of schedule, we can possibly amend the charter at that point to bring new features out of incubation?  Which is what I believe the text you proposed is saying.